File: C#/백준/Bronze/27245. Комната/Комната.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] aa = new int[2];
            for (int i=0; i<2; i++){
                aa[i] = int.Parse(Console.ReadLine());
            }
            int a = int.Parse(Console.ReadLine());
            if(aa.Min()>=a*2 && aa.Min()*2>=aa.Max()){
                Console.WriteLine("good");
            }
            else{
                Console.WriteLine("bad");
            }
        }
    }
}

File: Java/백준/Bronze/27245. Комната/Комната.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        int c = in.nextInt();
        int max = a>b?a:b;
        int min = a<b?a:b;
        if(min>=c*2 && max <= min*2){
            System.out.println("good");
        }
        else{
            System.out.println("bad");
        }
    }
}